code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -15,6 +15,7 @@
 		var/obj/item/clothing/suit/space/hardsuit/C = wear_suit
 		J = C.jetpack
 	if(istype(J) && J.allow_thrust(0.01, src))
+		pass_flags |= PASSTABLE

```

Comment:
because if the gravity goes out while someone is wearing a jetpack (which is probably going to be the case during gameplay) then it won't work